starter-code/src/functions-and-arrays.js

```diff
@@ -1,100 +1,442 @@
 // Find the maximum
-
+function maxOfTwoNumbers(num1,num2){
+  if (num1>num2){return num1}
+  else if (num2>num1){return num2}
+  else{return num1,num2}
+}
 // Finding Longest Word
 var words = [
-  'mystery',
-  'brother',
-  'aviator',
-  'crocodile',
-  'pearl',
-  'orchard',
-  'crackpot'
+  "mystery",
+  "brother",
+  "aviator",
+  "crocodile",
+  "pearl",
+  "orchard",
+  "crackpot"
```

Comment:
You didn't had to change. It's actually preferable to use single quotes in JS and double quotes in HTML